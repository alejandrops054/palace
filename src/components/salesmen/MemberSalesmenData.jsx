/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos Dev01
Comentarios: xxxxx
*/
import React from "react";
import {
  Input,
  Label,
  Row,
  Col,
  Button,
  InputGroup,
  InputGroupAddon,
  InputGroupText,
} from "reactstrap";

function MemberSalesmenData() {
  return (
    <>
      <h2>Member Salesmen</h2>
      <Row className="mb-2">
        <Col
          lg={{ size: 4 }}
          md={{ size: 5 }}
          sm={{ size: 6 }}
          xs={{ size: 6 }}
        >
          <Label>Number</Label>
          <Input type="number" />
        </Col>
      </Row>
      <Row className="mb-2">
        <Col
          lg={{ size: 4 }}
          md={{ size: 5 }}
          sm={{ size: 6 }}
          xs={{ size: 6 }}
        >
          <Label>Job</Label>
          <InputGroup>
            <Input type="text" />
            <InputGroupAddon addonType="append">
              <InputGroupText>
                <i className="fas fa-search"></i>
              </InputGroupText>
            </InputGroupAddon>
          </InputGroup>
        </Col>
      </Row>
      <Row className="mb-2">
        <Col
          lg={{ size: 4 }}
          md={{ size: 5 }}
          sm={{ size: 6 }}
          xs={{ size: 6 }}
        >
          <Label>Salesman</Label>
          <InputGroup>
            <Input />
            <InputGroupAddon addonType="append">
              <InputGroupText>
                <i className="fas fa-search"></i>
              </InputGroupText>
            </InputGroupAddon>
          </InputGroup>
        </Col>
      </Row>
      <Row className="mb-2">
        <Col>
          <Label>Name</Label>
          <Input type="text" />
        </Col>
      </Row>
      <Row className="mb-2">
        <Col
          lg={{ size: 4 }}
          md={{ size: 5 }}
          sm={{ size: 6 }}
          xs={{ size: 6 }}
        >
          <Label>Status</Label>
          <Input type="select">
            <option>Alta</option>
          </Input>
        </Col>
      </Row>
      <Row className="mb-2">
        <Col className="mt-5">
          <Button>Save</Button>
          <Button>Close</Button>
        </Col>
      </Row>
    </>
  );
}

export default MemberSalesmenData;
