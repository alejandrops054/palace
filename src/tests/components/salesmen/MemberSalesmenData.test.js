/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import MemberSalesmenData from '../../../components/salesmen/MemberSalesmenData'

describe('MemberSalesmenData component',() => {
    test('Debe mostrar snapshot', () => {
        
        const wrapper = shallow(<MemberSalesmenData/>)
        expect(wrapper).toMatchSnapshot();
    })
    
})
