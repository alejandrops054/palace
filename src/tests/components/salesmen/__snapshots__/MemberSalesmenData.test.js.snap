// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MemberSalesmenData component Debe mostrar snapshot 1`] = `
<Fragment>
  <h2>
    Member Salesmen
  </h2>
  <Row
    className="mb-2"
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <Col
      lg={
        Object {
          "size": 4,
        }
      }
      md={
        Object {
          "size": 5,
        }
      }
      sm={
        Object {
          "size": 6,
        }
      }
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
      xs={
        Object {
          "size": 6,
        }
      }
    >
      <Label
        tag="label"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        Number
      </Label>
      <Input
        type="number"
      />
    </Col>
  </Row>
  <Row
    className="mb-2"
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <Col
      lg={
        Object {
          "size": 4,
        }
      }
      md={
        Object {
          "size": 5,
        }
      }
      sm={
        Object {
          "size": 6,
        }
      }
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
      xs={
        Object {
          "size": 6,
        }
      }
    >
      <Label
        tag="label"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        Job
      </Label>
      <InputGroup
        tag="div"
      >
        <Input
          type="text"
        />
        <InputGroupAddon
          addonType="append"
          tag="div"
        >
          <InputGroupText
            tag="span"
          >
            <i
              className="fas fa-search"
            />
          </InputGroupText>
        </InputGroupAddon>
      </InputGroup>
    </Col>
  </Row>
  <Row
    className="mb-2"
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <Col
      lg={
        Object {
          "size": 4,
        }
      }
      md={
        Object {
          "size": 5,
        }
      }
      sm={
        Object {
          "size": 6,
        }
      }
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
      xs={
        Object {
          "size": 6,
        }
      }
    >
      <Label
        tag="label"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        Salesman
      </Label>
      <InputGroup
        tag="div"
      >
        <Input
          type="text"
        />
        <InputGroupAddon
          addonType="append"
          tag="div"
        >
          <InputGroupText
            tag="span"
          >
            <i
              className="fas fa-search"
            />
          </InputGroupText>
        </InputGroupAddon>
      </InputGroup>
    </Col>
  </Row>
  <Row
    className="mb-2"
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <Col
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Label
        tag="label"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        Name
      </Label>
      <Input
        type="text"
      />
    </Col>
  </Row>
  <Row
    className="mb-2"
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <Col
      lg={
        Object {
          "size": 4,
        }
      }
      md={
        Object {
          "size": 5,
        }
      }
      sm={
        Object {
          "size": 6,
        }
      }
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
      xs={
        Object {
          "size": 6,
        }
      }
    >
      <Label
        tag="label"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        Status
      </Label>
      <Input
        type="select"
      >
        <option>
          Alta
        </option>
      </Input>
    </Col>
  </Row>
  <Row
    className="mb-2"
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <Col
      className="mt-5"
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Button
        color="secondary"
        tag="button"
      >
        Save
      </Button>
      <Button
        color="secondary"
        tag="button"
      >
        Close
      </Button>
    </Col>
  </Row>
</Fragment>
`;
